pkg/handlers/v1/dependencycheck.go

```diff
@@ -0,0 +1,18 @@
+package v1
+
+import (
+	"context"
+
+	"github.com/asecurityteam/nexpose-asset-producer/pkg/domain"
+)
+
+// DependencyCheckHandler takes in a domain.DependencyChecker to check external dependencies
+type DependencyCheckHandler struct {
+	AssetFetcher domain.AssetFetcher
```

Comment:
Task: Please cleanly separate the `DependencyChecker` from the `AssetFetcher`.

Reply:
done